### PR TITLE
First simple attempt registration

### PR DESCRIPTION
**Pull request by shnubu**
Open for any critics i am frontend noob, log in is not realized yet

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...tiveDpr' is defined but never used.","'Loader' is defined but never used.","'PerspectiveCamera' is defined but never used.","'Paper' is defined but never used.","'Snackbar' is defined but never used.","'SvgIcon' is defined but never used.","'Switch' is defined but never used.","'Alert' is defined but never used.","'VolumeUpIcon' is defined but never used.","'VolumeOffIcon' is defined but never used.","'mdiTabletCellphone' is defined but never used.","'mdiVideo3d' is defined but never used.","'mdiVideo3dOff' is defined but never used.","'Icon' is defined but never used.","'classes' is assigned a value but never used.","'useHistory' is defined but never used.","'MuiAlert' is defined but never used.","'Backdrop' is defined but never used.","'CircularProgress' is defined but never used.","'TextareaAutosize' is defined but never used.","'TextField' is defined but never used.","'clsx' is defined but never used.","'Link' is defined but never used.","'IconButton' is defined but never used.","'TwitterIcon' is defined but never used.","'MDBIcon' is defined but never used.","'MDBContainer' is defined but never used.","'Fade' is defined but never used.","no-global-assign","no-unsafe-negation"]
```
Comment (Hialus) [inline .eslintcache:1]: Ich bin kein Experte was React angeht, aber da diese Datei cache im Namen hat würde ich sagen, dass diese Datei zur .gitignore Datei hinzugefügt werden.

**Review by Hialus — CHANGES_REQUESTED** [review]
The .idea Folder and .iml files should be placed in the .gitignore file as well.

**Review by Hialus — APPROVED** [review]

**Comment by Hialus** [comment]
From my side everything looks good. But it would be great if someone with more React experience could take a look as well.

**Comment by MarcZiegler** [comment]
Looks good to me!

**Pull request merged**